packages/fast-documentation/build/clean.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Utility for cleaning directories.
+ * Usage: node build/clean.js %path%
+ */
+const path = require("path");
+const rimraf = require("rimraf");
+const argv = require("yargs").argv;
+
+/**
+ * All paths passed to the clean script
+ */
+const paths = argv._;
+
+/**
+ * Function to remove a given path
+ */
+function cleanPath(cleanPath) {
+    const removePath = path.resolve(process.cwd(), cleanPath);
+    rimraf(removePath, () => {
+        console.log(removePath, "cleaned");
+    });
+}
+
+/**
+ * Clean all paths
+ */
+if (Array.isArray(paths)) {
+    paths.forEach(cleanPath);
+}
```

Comment:
I think we have a clean file in the root - can we use that rather than having this be an outlier?

Reply:
change made